build.sbt

```diff
@@ -333,7 +375,9 @@ def multiScalaDirectories(root: File, scalaVersion: String) = {
 }
 
 val mtagsSettings = List(
-  crossScalaVersions := V.supportedScalaVersions,
+  crossScalaVersions := {
+    V.supportedScalaVersions ++ V.nightlyScala3Versions
```

Comment:
With this, why would we need `configureMtagsScalaVersionDynamically` ?

Reply:
This is needed to publish mtags for nightly versions.
So at the release moment of for snapshot supported scala versions will be: all explicitly defined scala versions + up to 5 last nightly